server/service/src/sync/integrate_document.rs

```diff
@@ -12,12 +12,12 @@ use crate::programs::{
     program_enrolment::program_schema::SchemaProgramEnrolment,
 };
 
-pub(crate) fn upsert_document(
+pub(crate) fn sync_upsert_document(
     con: &StorageConnection,
     document: &Document,
-    is_sync_update: bool,
 ) -> Result<(), RepositoryError> {
-    DocumentRepository::new(con).insert(document, is_sync_update)?;
+    // TODO comment why only insert here
```

Comment:
@clemens-msupply I can add the extra comment. I would assume that documents arriving via sync would always be unique ? Btw what is uniqueness of document ? Also if there is a duplicate, this would be ok right ? This integration will fail, recorded and sync carries on (a bit like FK error), and this shouldn't' cause any inconsistencies down the line right ?

Reply:
Yes if there is already an existing document, then it is the same that we are going to integrate. So it shouldn't actually be a problem in sync. Think I can replace the insert with an upsert method for consitency though.